src/qrl/core/txs/SlaveTransaction.py

```diff
@@ -7,7 +7,11 @@
 
 
 class SlaveTransaction(Transaction):
-
+    """
+    This registers one XMSS tree as the slave of another XMSS tree,
```

Comment:
It is worth to mention that in SlaveTransaction one can register maximum upto 100 slaves per transaction.